iri.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
 		}
 
 		const iri = parse(value);
-		if ((iri.reference === 'absolute' || iri.reference === 'uri') && iri.scheme && iri.host) {
+		if ((iri.reference === 'absolute' || iri.reference === 'uri') && iri.scheme && typeof iri.host !== 'undefined') {
```

Comment:
iri.host is always defined, so this check doesn't make a lot of sense. This should probably be an exclusive exception jjust for the file protocol and require path in this case.

```suggestion
		if ((iri.reference === 'absolute' || iri.reference === 'uri') && iri.scheme && (iri.host || (iri.scheme === 'file' && iri.path))) {
```